### ISSUE TITLE
Issues upgrading from 1.7 to 1.8

### ISSUE THREAD

**Issue by tmerten** [geospoc/v-mapbox#558]
Hi there,

we did some version changes in a project and (I think) `v-mapbox` responded a little unexcepted. As version changes go, a lot of things were fiddled with and I hope I won't forget anything telling the story:

* It started by removing a depenency from a `vue 2.6` project and removing and rebuilding the `yarn.lock` file
  * the project was running `v-mapbox 1.7.x` and `mapbox-gl 2.2.x` without problems before (although I am unsure if that combination was ever recommended ;) )
  * our package json has `"v-mapbox": "^1.7.0"` and after a `yarn install` the `yarn.lock` file shows that `v-mapbox` was updated to `1.8.2`
```
v-mapbox@^1.7.0:
  version "1.8.2"
  resolved "https://registry.yarnpkg.com/v-mapbox/-/v-mapbox-1.8.2.tgz# [...]
  dependencies:
    map-promisified "^0.4.0"
```
  * at the same time `1.8` seems to introduce breaking changes (e.g. requiring the `container` prop and depending on the composition API) thus the update was unexpected
     * by the way the `container` prop [is required but has a `default value` still](https://github.com/geospoc/v-mapbox/blob/28c809653e1135162e859c8328cf925de4a54d0b/src/components/map/options.ts#L62) 
* after understanding what happened, we added the dependency `@vue/composition-api": "^1.3.0"` and the `container` prop
* now are now stuck with an error saying `TypeError: this.templateRefs is undefined` 
  * we changed the mapbox gl dependency to `"mapbox-gl": ">=1 <2"` but this did not help either
  * do we need to add a mapbox instance?
* of course downgrading back to `"v-mapbox": "1.7.3"` solves the issue but I am happy to help and give more infos

TLDR; I think a `^1.7.0` should not upgrade to `1.8` and a note about the breaking changes and how to resolve those might help.

(please be so kind and do not point me immediately to the part of the documentation where this issue is addressed and which I overlooked ;) )
